lib/posix/posix_linux_amd64.nim

```diff
@@ -244,8 +241,6 @@ type
     st_atim*: Timespec   ## Time of last access.
     st_mtim*: Timespec   ## Time of last data modification.
     st_ctim*: Timespec   ## Time of last status change.
-    reserved: array[3, clong]
```

Comment:
This must stay for binary compatibility. NLVM needs it.

Reply:
Ah, I was trying to figure out why this was here. Is there a way to hide `reserved` from iterators?

Reply:
Currently not but we need some new annotation like ``reserved {.padding.}``

Reply:
=> was discussed in https://github.com/nim-lang/RFCs/issues/205 propoosal 4 {.hidden.}

Reply:
this broke `nlvm`